projects/media-viewer/src/lib/media-viewer.module.ts

```diff
@@ -48,7 +67,11 @@ import {RouterModule} from '@angular/router';
 ],
   providers: [
     PdfJsWrapperFactory,
-    CommentService
+    CommentService,
+    {
+      provide: RouterStateSerializer,
+      useClass: CustomSerializer
+    },
```

Comment:
do we need these right now? we can always add them later when we need them
although, given that this is a library, are we ever likely to be doing routing